src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -651,21 +651,24 @@ function disableRow(elem, row) {
     $tdElements.parent().hide();
 }
 
-function feedbackRecipientUpdateVisibilityOptions(elem) {
-    elem = $(elem);
-    if (elem.val() === 'OWN_TEAM' || elem.val() === 'TEAMS' ||
-            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS') {
-        enableRow(elem, 1);
-        disableRow(elem, 3);
+function feedbackRecipientUpdateVisibilityOptions(recipientType) {
+    recipientType = $(recipientType);
```

Comment:
@Sumei1009, prefix this variable with <code>$</code>. Make this a local variable.